pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -377,6 +382,26 @@ func updateStatusFromPod(taskRun *v1alpha1.TaskRun, pod *corev1.Pod) {
 		// update tr completed time
 		taskRun.Status.CompletionTime = &metav1.Time{Time: time.Now()}
 	}
+
+	updateTaskRunResourceResult(taskRun, pod, resourceLister, kubeclient, logger)
+}
+
+func updateTaskRunResourceResult(taskRun *v1alpha1.TaskRun, pod *corev1.Pod, resourceLister listers.PipelineResourceLister, kubeclient kubernetes.Interface, logger *zap.SugaredLogger) {
+	if resources.TaskRunHasOutputImageResource(resourceLister.PipelineResources(taskRun.Namespace).Get, taskRun) && taskRun.IsSuccessful() {
+		for _, container := range pod.Spec.Containers {
+			if strings.HasPrefix(container.Name, imageDigestExporterContainerName) {
+				req := kubeclient.CoreV1().Pods(taskRun.Namespace).GetLogs(pod.Name, &corev1.PodLogOptions{Container: container.Name})
+				logContent, err := req.Do().Raw()
```

Comment:
hm would it be a bit cleaner to do a `req.Stream()` here (and defer closing the stream)?

Reply:
wouldn't it make the code more verbose :thinking: (I am "only" guessing that `Ram()` does that for us in one shot, no ?)

Reply:
its expected to be about a line or two, so not sure worth the complexity

Reply:
kk